Header header
geometry_msgs/Vector3 position
geometry_msgs/Vector3 velocity
Mode posMode
float64 yaw
